docs/readthedocs/source/doc/Nano/Howto/quantize_pytorch_inference_inc.md

```diff
@@ -0,0 +1,72 @@
+# Quantize PyTorch Model for Inference using Intel Neural Compressor
+
+With Intel Neural Compressor (INC) as quantization engine, you can apply `Trainer.quantize` API to realize post-training quantization on your PyTorch `nn.Module`. It only takes a few lines.
+
+Let's take an [ResNet-18 model](https://pytorch.org/vision/main/models/generated/torchvision.models.resnet18.html) pretrained on ImageNet dataset and finetuned on [OxfordIIITPet dataset](https://pytorch.org/vision/main/generated/torchvision.datasets.OxfordIIITPet.html) as an example:
+
+```python
+from torchvision.models import resnet18
+
+model = resnet18(pretrained=True)
+_, train_dataset, val_dataset = finetune_pet_dataset(model)
+```
+> _The full definition of function_ `finetune_pet_dataset` _could be found_ [_here_](https://github.com/intel-analytics/BigDL/blob/1122517e0f919d7880e8f9848a476ce889ed7183/python/nano/tutorial/inference/pytorch/pytorch_quantization.py#L33).
+
+Then we set it in evaluation mode:
+
+```python
+model.eval()
+```
```

Comment:
same here.

Reply:
shall we move this step to `Trainer.trace`? @TheaperDeng

Reply:
yeah, we need to do this.